docs/README.md

```diff
@@ -0,0 +1,155 @@
+# Yielding Transaction Pattern Library (ytxp-lib)
+
+The Yielding Transaction Pattern is pretty simple.
+
+- A Yielding Script only succeeds if it can find evidence that an _authorized_ "Yielded To" script also succeeded.
+- The _authorization_ is determined by examination of the script context.
+- The _authorization_ is checked by:
+  - Indexing the reference inputs to find evidence that a "Yielded To" script containing a valid `YieldedToSTCS` is being referenced
```

Comment:
I'm not sure I understand this line.

What is the `YieldedToSTCS`? 

Do you mean to say something like "Indexing the reference inputs to find evidence that `AuthorizedScriptUTxO` is being referenced, such that the UTxO carries a token with the "authorizedScriptSTCS" in its `Value`"?